src/main/java/com/direwolf20/buildinggadgets/client/events/EventTooltip.java

```diff
@@ -30,47 +41,37 @@
 
 @Mod.EventBusSubscriber(modid = Reference.MODID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
 public class EventTooltip {
-
+    private static final Comparator<Multiset.Entry<IUniqueObject<?>>> ENTRY_COMPARATOR = Comparator
+            .<Multiset.Entry<IUniqueObject<?>>, Integer>comparing(Entry::getCount)
+            .reversed()
+            .thenComparing(e -> e.getElement().getObjectRegistryName());
     private static final int STACKS_PER_LINE = 8;
     private static RemoteInventoryCache cache = new RemoteInventoryCache(true);
 
-    public static void setCache(Multiset<UniqueItem> cache) {
+    public static void setCache(Multiset<com.direwolf20.buildinggadgets.common.util.tools.UniqueItem> cache) {
```

Comment:
There used to be both UnqiueItems in that class, that messed it up... Should be ok to use an import now.

Reply:
@MichaelHillcox this actually needs to use the full name, because RemoteInventories aren't updated to the new System and therefore still need the old UniqueItem.

Reply:
Updating that would take to much time for now, therefore I'll leave this